xtask/codegen/src/css_kinds_src.rs

```diff
@@ -92,6 +92,7 @@ pub const CSS_KINDS_SRC: KindsSrc = KindsSrc {
         "var",
         "font-palette-values",
         "font-face",
+        "url",
```

Comment:
Move please `url` before `font-face`.
otherwise we have to modify the `is_contextual_keyword` function.